README.md

```diff
@@ -1,6 +1,6 @@
 # RankDB
 
-[![Build Status](https://travis-ci.com/Vivino/rankdb.svg?branch=master)](https://travis-ci.com/Vivino/rankdb)
+[![Build Status](https://app.travis-ci.com/Vivino/rankdb.svg?branch=master)](https://app.travis-ci.com/github/Vivino/rankdb)

```

Comment:
No longer relevant.

Reply:
Yeah, i'll update links when GA is going to be configured and working.